tasties_app/tests/test_ingredient.py

```diff
@@ -0,0 +1,59 @@
+import pytest as pytest
+
+from tasties_app.models import Ingredient, Recipe, Category
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def create_data():
+    """
+    Creates 2 users, 2 recipes, and an ingredient,
+    and adds them to the database.
+    """
+    user1 = User.objects.create_user('john', 'password')
+    user2 = User.objects.create_user('paul', 'password')
+    c1 = Category.objects.create(category_name='Breakfast')
+    c1.save()
+    test_data = [
+        ('Test Recipe1', user1, 'Test Description1', 'Test Directions1', 1, 'test_picture1'),
+        ('Test Recipe2', user2, 'Test Description2', 'Test Directions2', 2, 'test_picture2'),
+    ]
+    for title, aid, desc, dirs, mints, pic in test_data:
+        recipe = Recipe(title=title, author_id=aid, description=desc, directions=dirs,
+                        minutes_to_make=mints, recipe_picture=pic)
+        recipe.save()
+        recipe.categories.add(c1)
+        Ingredient(recipe_id=recipe, amount=1.0, measurement_unit='WHOLE',
+                   description='test ingredient').save()
+
+
+class TestIngredientModel:
+    """
+    Tests the Ingredient model:
+        - Creation of an ingredient
+        - Editing of an ingredient
+    """
+    @pytest.mark.django_db
+    def test_false(self):
+        assert False
+
+    @pytest.mark.django_db
+    def test_create_ingredient(self, create_data):
+        ingredient_set = Ingredient.objects.all()
+        ingredient1 = ingredient_set[0]
+        assert ingredient1.recipe_id.title == 'Test Recipe1'
+        assert ingredient1.description == 'test ingredient'
+        assert ingredient1.amount == 1.0
+        assert ingredient1.measurement_unit == 'WHOLE'
+        assert ingredient1.recipe_id.author_id.username == 'john'
+        assert ingredient1.recipe_id.categories.all()[0].category_name == 'Breakfast'
+
+    @pytest.mark.django_db
+    def test_edit_ingredient(self, create_data):
+        ingredient_set = Ingredient.objects.all()
+        ingredient1 = ingredient_set[0]
+        ingredient1.edit_info(2.0, "FLOZ", "test2")
+        assert ingredient1.amount == 2.0
+        assert ingredient1.measurement_unit == "FLOZ"
+        assert ingredient1.description == "test2"
+        
```

Comment:
In this line flake8 say that you have blank line that contains whitespace

Reply:
Thanks, Fixed it.
Now waiting for #43 to allow pytest tests to pass.